teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < Person
+  def initialize(specialization)
+    super()
```

Comment:
- [ ] to make this work please pass all the arguments from the initialize method in the Person class in this initialize method as well and add them to the super as arguments as well.